src/plopp/backends/pythreejs/canvas.py

```diff
@@ -56,7 +47,9 @@ def __init__(
         self._title_text = title
         self._title = self._make_title()
         width, height = self.figsize
-        self._raw_user_camera = camera if camera is not None else {}
+        self._raw_user_camera = Camera(
+            **(camera if camera is not None else {})
+        ).asdict()
```

Comment:
Why unpack to dict and put it back into `Camera`? Why is the `asdict` required?

Reply:
This made some filtering below easier, but I agree that it's a little weird. I'll refactor, depending on what we decide above.